Orange/widgets/data/owfeaturestatistics.py

```diff
@@ -500,7 +500,7 @@ def data(self, index, role):
             if role == Qt.DisplayRole:
                 if isinstance(attribute, DiscreteVariable):
                     output = self._center[row]
-                    if not np.isnan(output):
```

Comment:
What about `else`? Should `output` be set to `""` in this case? `np.nan` is properly handled below with `if output==output: output = ""`, but this won't work if an attribute has no values and `self._center[row]` is ... whatever it is in this case (probably `0`).